packages/mdc-floating-label/_mixins.scss

```diff
@@ -55,16 +57,20 @@
   }
 }
 
-@mixin mdc-floating-label-float-position($positionY, $positionX: 0%, $scale: .75) {
+@mixin mdc-floating-label-float-position($positionY, $positionX: 0%, $scale: .75, $query: mdc-feature-all()) {
```

Comment:
I copied this change from #5287 since I needed it here and I didn't want to wait for the other PR to be merged. I'll rebase if necessary.